packages/core/src/form/reatomForm.test.ts

```diff
@@ -522,6 +522,47 @@ test('triggering schema validation only for one field', async () => {
   expect(form.fields.age.validation().error).toBe('must be minimum 18')
 })
 
+test('correct handling of side errors from schema', async () => {
+  const INVARIANT_ERR_MSG = 'value "from" of should be less than "to" value';
+
+  const form = reatomForm({
+    from: 0,
+    to: 10
```

Comment:
Nice example, add it to the docs
```suggestion
    min: 0,
    max: 10
```